cmake/linux/package_linux.sh.in

```diff
@@ -40,8 +39,15 @@ function skipped {
 	echo -e "   ${PLAIN}[${YELLOW}skipped${PLAIN}] ${1}"
 }
 
+# Run a command silently, but print output if it fails
+function try_quiet {
+    output="$("$@" 2>&1)" && return
+    echo "$output"
```

Comment:
How about printing the failed command as well?

Reply:
I tried but it got messy because

    try_quiet "$LINUXDEPLOYQT" "$DESKTOPFILE" $executables -bundle-non-qt-libs -verbose=$VERBOSITY $STRIP

that line expands to like 1500+ characters